### PR TITLE
chore: stabilisation rename keypair

### PR DESCRIPTION
**Pull request by Valentina1133**
- Added attempts
- Reworked wait_until_appears method for add account popup

https://ci.status.im/job/status-desktop/job/e2e/job/manual/1939/allure/#suites/e5028aa5a5a84c948bb9af2c7ccc9fec/1754ae5a1e64adf6/

### CODE REVIEW

File: gui/components/wallet/wallet_account_popups.py [hunk 1]
```diff
@@ -196,14 +207,20 @@ def __init__(self):
...
+            names.mainWallet_AddEditAccountPopup_MasterKey_GenerateSeedPhraseOption)
         self._seed_phrase_12_words_button = Button(names.mainWallet_AddEditAccountPopup_12WordsButton)
         self._seed_phrase_18_words_button = Button(names.mainWallet_AddEditAccountPopup_18WordsButton)
         self._seed_phrase_24_words_button = Button(names.mainWallet_AddEditAccountPopup_24WordsButton)
         self._seed_phrase_word_text_edit = TextEdit(names.mainWindow_statusSeedPhraseInputField_TextEdit)
         self._seed_phrase_phrase_key_name_text_edit = TextEdit(
             names.mainWallet_AddEditAccountPopup_ImportedSeedPhraseKeyName)
 
+    @allure.step('Wait until appears {0}')
+    def wait_until_appears(self, timeout_msec: int = configs.timeouts.UI_LOAD_TIMEOUT_MSEC):
```
Comment (anastasiyaig) [inline gui/components/wallet/wallet_account_popups.py:220]: why not using / reworking https://github.com/status-im/desktop-qa-automation/blob/3f452e86bb47fcd026639ca5b0c1d21e47b254c0/gui/components/wallet/wallet_account_popups.py#L51 ?

I use it  in https://github.com/status-im/desktop-qa-automation/blob/2d630d245c2ec84f59f6d0707d8afc02bf552fa3/gui/screens/settings_wallet.py, it seem to be working, we can rework it by checking if the needed button is there, but i also wanted to check if that is the modal we need (Add account or Edit account, thats why i added verification by title for now)

Comment (Valentina1133) [reply gui/components/wallet/wallet_account_popups.py:220]: Ah! Ok, I will use it. It's just because this PR was created before your changes :)

Comment (Valentina1133) [reply gui/components/wallet/wallet_account_popups.py:220]: No, it's different. I will use it for sure!

Comment (Valentina1133) [reply gui/components/wallet/wallet_account_popups.py:221]: It was done! Forgot to say )

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**